src/coreclr/vm/arm64/thunktemplates.S

```diff
@@ -123,11 +123,15 @@ LEAF_END_MARKED CallCountingStubCodeTemplate, _TEXT
         ldr x10, DATA_SLOT(StubPrecode, Target)
         ldr x12, DATA_SLOT(StubPrecode, SecretParam)
         br x10
+        brk     0xf000      // Stubs need to be 24-byte in size to allow for the data to be 3 pointers
+        brk     0xf000      // Stubs need to be 24-byte in size to allow for the data to be 3 pointers
+        brk     0xf000      // Stubs need to be 24-byte in size to allow for the data to be 3 pointers
     LEAF_END_MARKED StubPrecodeCode\STUB_PAGE_SIZE
 
     LEAF_ENTRY FixupPrecodeCode\STUB_PAGE_SIZE
         ldr x11, DATA_SLOT(FixupPrecode, Target)
         br  x11
+        dmb ishld
```

Comment:
The same here.

Reply:
No, that's still necessary. We're not flushing around creation of `FixupPrecode`, since it's expensive and I was able to come up with an alternative approach for that.